src/lib/uTop.ml

```diff
@@ -196,6 +208,7 @@ let discard_formatters pps f =
    +-----------------------------------------------------------------+ *)
 
 type location = int * int
+type lines = int * int
```

Comment:
Can this be a record instead? This will simplify reading the code.

Reply:
Ack, I used the tuple for consistency with location. 100% agreed record is the right way to go!